### PR TITLE
V8 composing

### PR DESCRIPTION
**Pull request by warrenbuckley**
Currently, this is a draft/WIP document for V8 Composers/Components & Collections

### CODE REVIEW

**Comment by warrenbuckley** [comment]
Paging @zpqrtbnk for help with some of the `TODOs` mostly about trying to explain the fundamental concepts with the correct wording.

Stephan could you please do a sanity check over this & make sure I haven't rambled or given out any misleading information :)

**Comment by warrenbuckley** [comment]
@bergmania are you able to sanity check this from a tech point of view and then we can pass over to docs team to grammar check my mess of words :-P

File: Implementation/Composing/index-v8.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+        }
+    }
+}
+```
+
+
+## Attributes
+Umbraco has some useful C# attributes to decorate your composer classes or Types used in collections.
+
+### [ComposeBefore]` and `[ComposeAfter]`
```
Comment (jmayntzhusen) [inline Implementation/Composing/index-v8.md]:
```suggestion
### `[ComposeBefore]` and `[ComposeAfter]`
```

File: Implementation/Composing/index-v8.md [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+keywords: composing composers components runtime boot booting v8 version8
+versionFrom: 8.0.0
+---
+
+# Composing
+With Umbraco V8+ you are now able to customise how your Umbraco application runs and boots by composing different components of the system. You may wish to add your own customisations on how Umbraco works or alternatively disable or remove specific functions in Umbraco. You can achieve this with composers & components.
+
+
+### Changing Umbraco
+Below is a simple sample of changing the Umbraco application to allow Spotify URLs to be used as OEmbed in the Grid and Rich Text Editors inside Umbraco. As the collection for OEmbedProviders is not typed scanned, then you will see that we need to explicitly add to the collection.
```
Comment (jmayntzhusen) [inline Implementation/Composing/index-v8.md]:
```suggestion
Below is a sample of changing the Umbraco application to allow Spotify URLs to be used as OEmbed in the Grid and Rich Text Editors inside Umbraco. As the collection for OEmbedProviders is not typed scanned, then you will see that we need to explicitly add to the collection.
```

**Pull request merged**